demos/browser/server.js

```diff
@@ -22,14 +23,16 @@ const indexPage = fs.readFileSync(indexPagePath);
 // Create ans AWS SDK Chime object. Region 'us-east-1' is currently required.
 // Use the MediaRegion property below in CreateMeeting to select the region
 // the meeting is hosted in.
-const chime = new AWS.Chime({ region: 'us-east-1' });
-const sts = new AWS.STS({ region: 'us-east-1' })
+const currentRegion = 'us-east-1';
+const sts = new AWS.STS({ region: currentRegion });
 
 // Set the AWS SDK Chime endpoint. The global endpoint is https://service.chime.aws.amazon.com.
 const endpoint = process.env.ENDPOINT || 'https://service.chime.aws.amazon.com';
-console.info('Using endpoint', endpoint);
 
-chime.endpoint = new AWS.Endpoint(endpoint);
+//Set useGlobalEndpoint to true to use global endpoint
+//Setting it false will use regional Chime Meetings Endpoints based upon region.
+const useGlobalEndpoint = false;
```

Comment:
The vars `endpoint` and `useGlobalEndpoint` are in semantic conflict since the default endpoint there is the global endpoint, despite `useGlobalEndpoint` is default `false`

Reply:
updated and removed useGlobalEndpoint